src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -37,10 +37,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar"
-             Link="jars\%(Filename)%(Extension)"
-             Pack="true"
-             PackagePath="jars\%(Filename)%(Extension)" />
+    <Content Include="..\..\scala\microsoft-spark-*\target\microsoft-spark-*.jar" Link="jars\%(Filename)%(Extension)" Pack="true" PackagePath="jars\%(Filename)%(Extension)" />
```

Comment:
On one hand, this change is unnecessary and should be reverted. On the other hand, I don't like fighting VS....

Reply:
Read my mind :)